Update configure-endpoints-sccm.md (#13143)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/configure-endpoints-sccm.md

For more information about System Center Configuration Manager Compliance, see [Introduction to compliance settings in System Center 2012 R2 Configuration Manager](/previous-versions/system-center/system-center-2012-R2/gg682139\(v=technet.10\)).

### Onboard Windows devices using Microsoft Configuration Manager

### Collection creation

To onboard Windows devices with Microsoft Configuration Manager, the deployment can target an existing collection or a new collection can be created for testing.

> Onboarding using tools such as Group policy or manual method doesn't install any agent on the system.

Reviewer: ```suggestion
Onboarding using tools such as Group Policy or a manual method doesn't install any agents on the system.
```

> Within the Microsoft Configuration Manager, console the onboarding process will be configured as part of the compliance settings within the console.

Reviewer: ```suggestion
Within the Microsoft Configuration Manager console, the onboarding process will be configured as part of the compliance settings within the console.
```

Any system that receives this required configuration maintains that configuration for as long as the Configuration Manager client continues to receive this policy from the management point.

> Follow the steps below to onboard endpoints using Microsoft Configuration Manager.

Reviewer: ```suggestion
Follow these steps to onboard endpoints using Microsoft Configuration Manager:
```

> 1. In Microsoft Configuration Manager console, navigate to **Assets and Compliance \> Overview \> Device Collections**.

Reviewer: ```suggestion
1. In the Microsoft Configuration Manager console, navigate to **Assets and Compliance \> Overview \> Device Collections**.
```

    :::image type="content" source="images/configmgr-device-collections.png" alt-text="The Microsoft Configuration Manager wizard1" lightbox="images/configmgr-device-collections.png":::

> 2. Right select **Device Collection** and select **Create Device Collection**.

Reviewer: ```suggestion
2. Select and hold (or right-click) **Device Collection** and select **Create Device Collection**.
```

    :::image type="content" source="images/configmgr-create-device-collection.png" alt-text="The Microsoft Configuration Manager wizard2" lightbox="images/configmgr-create-device-collection.png":::

3. Provide a **Name** and **Limiting Collection**, then select **Next**.

    :::image type="content" source="images/configmgr-limiting-collection.png" alt-text="The Microsoft Configuration Manager wizard3" lightbox="images/configmgr-limiting-collection.png":::

4. Select **Add Rule** and choose **Query Rule**.

    :::image type="content" source="images/configmgr-query-rule.png" alt-text="The Microsoft Configuration Manager wizard4" lightbox="images/configmgr-query-rule.png":::

> 5. Select **Next** on the **Direct Membership Wizard** and select on **Edit Query Statement**.

Reviewer: ```suggestion
5. Select **Next** on the **Direct Membership Wizard** and then select **Edit Query Statement**.
```

    :::image type="content" source="images/configmgr-direct-membership.png" alt-text="The Microsoft Configuration Manager wizard5" lightbox="images/configmgr-direct-membership.png":::

6. Select **Criteria** and then choose the star icon.

    :::image type="content" source="images/configmgr-criteria.png" alt-text="The Microsoft Configuration Manager wizard6" lightbox="images/configmgr-criteria.png":::

> 7. Keep criterion type as **simple value**, choose whereas **Operating System - build number**, operator as **is greater than or equal to** and value **14393** and select on **OK**.

Reviewer: ```suggestion
7. Keep criterion type as **simple value**, choose whereas **Operating System - build number**, operator as **is greater than or equal to** and value **14393**, and select **OK**.
```

    :::image type="content" source="images/configmgr-simple-value.png" alt-text="The Microsoft Configuration Manager wizard7" lightbox="images/configmgr-simple-value.png":::

8. Select **Next** and **Close**.

    :::image type="content" source="images/configmgr-membership-rules.png" alt-text="The Microsoft Configuration Manager wizard8" lightbox="images/configmgr-membership-rules.png":::

9. Select **Next**.

    :::image type="content" source="images/configmgr-confirm.png" alt-text="The Microsoft Configuration Manager wizard9" lightbox="images/configmgr-confirm.png":::

> After completing this task, you now have a device collection with all the Windows endpoints in the environment.
> 

Reviewer: ```suggestion
After completing this task you have a device collection with all the Windows endpoints in the environment.
```
